SimplifiedR2.xcworkspace/contents.xcworkspacedata

```diff
@@ -4,16 +4,4 @@
    <FileRef
       location = "group:Simplified.xcodeproj">
    </FileRef>
-   <FileRef
```

Comment:
I would keep these in the workspace. Even if we don't need the NYPL forks, it's still useful to have a workspace for doing R2 integration. We will just need to update the remote in our local clones (or reclone them in the same location).

Reply:
Can cloning and adding one of these repositories only when needed be a better solution, like we do with NYPLAudiobookToolkit? We won't be doing anything in the repositories anymore; over time cloned local repositories' branch can become different to one we have in `Cartfile.resolved`, causing confusion to developers not working with R2 features.

Reply:
> We won't be doing anything in the repositories anymore; 

you're optimistic :) since R2 is still sort of in flux, there might be some debugging that we have to do inside R2. For that I found it easier to use the `R2dev` target which uses these repo links. So I'd like to keep it at the very least until we ship R2.

> over time cloned local repositories' branch can become different to one we have in `Cartfile.resolved`, causing confusion to developers not working with R2 features.

right, sure. BTW this is not adding a requirement to developers not working on R2: everyone can still keep using the main "SimplyE" target as explained in the README. That target uses the Carthage links just like NYPLAudiobookToolkit.

Reply:
Added references to SimplifiedR2.xcworkspace